localstack/utils/testutil.py

```diff
@@ -605,6 +605,9 @@ def get_log_events(function_name, delay_time):
             or "START" in raw_message
             or "END" in raw_message
             or "REPORT" in raw_message
+            # necessary until tail is updated in docker images. See this PR:
+            # http://git.savannah.gnu.org/gitweb/?p=coreutils.git;a=commitdiff;h=v8.24-111-g1118f32
+            or "tail: unrecognized file system type 0x794c7630" in raw_message
```

Comment:
nit: Wondering if we should simply filter on `"tail: unrecognized file system type"`, or is the type identifier `0x794c7630` always the same?

Reply:
The type identifier should be for overlayFS and should stay the same. We can ditch it though, for future file systems :)